### PR TITLE
Update to v 2.4.2:

### PR DESCRIPTION
**Pull request by akabanovs**
- version bumped up and sha256 updated
 - removed overspecified info in the 'about' section to satisfy linter

### CODE REVIEW

**Comment by akabanovs** [comment]
There is a later version 2.5.0 available but s3fs requires this one. No other packages use aiobotocore.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,10 +8,10 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: I do not think this is needed

Comment (akabanovs) [reply recipe/meta.yaml]: yes, removed.

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**